featurebyte/api/event_table.py

```diff
@@ -471,7 +471,7 @@ def create_new_feature_job_setting_analysis(
         return analysis
 
     @typechecked
-    def initialize_default_feature_job_setting(self) -> None:
+    def initialize_default_feature_job_setting(self, min_featurejob_period: int = 3600) -> None:
```

Comment:
Would it be cleaner to name this argument `min_period` or `min_period_seconds`? In the `FeatureJobSetting` class, the period is a string - maybe we want to keep it consistent here.